### ISSUE TITLE
Is this a placeholder repository for Microsoft.Azure.Cosmos.Table?

### ISSUE THREAD

**Issue by liquidsnk** [Azure/azure-cosmos-table-dotnet#9]
Corresponding to this nuget package https://www.nuget.org/packages/Microsoft.Azure.Cosmos.Table?

If so, any idea on when the source would be available? Or am I looking in the wrong place?

**Comment by MisinformedDNA** [comment]
Alternate question: Is the source _ever_ going to be available?

**Comment by ThomasWeiss** [comment]
@MisinformedDNA Yes, and we apologize for the delay, but the architecture of the V1 of this SDK just made it impractical to open-source its content. V2 has been re-designed by taking a dependency on our SQL API SDK, and we're in the process of making it open-source.

**Comment by GeertVL-zz** [comment]
Any update on making the SDK opensource. 
And why is it not a part of the large repository of the Azure SDK?

**Comment by HansOlavS** [comment]
The 2.0 SDK is still in preview, even now after almost a year. Why isn't there any update on this? It seems to me that Microsoft should've open-sourced this a year ago and it would have been in release a long long time ago. The point of open-source is to get important stuff like this SDK out quicker and better. It's free labor - I honestly don't understand why this isn't done yet.

**Comment by PaulCheng** [comment]
CosmosDB team is working with Azure SDK folks defining the best path forward. Hopefully it will be finalized soon.
